versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java

```diff
@@ -303,7 +313,16 @@ void preventNamespaceDeletionWithChildren(boolean childNamespace) throws Excepti
                     Optional.empty(),
                     fromMessage("try delete ns"),
                     singletonList(Delete.of(ns.toContentKey()))))
-        .isInstanceOf(ReferenceConflictException.class);
+        .hasMessage("The namespace 'ns' would be deleted, but cannot, because it has children.")
```

Comment:
I wonder if we couldn't simplify the message to just `namespace 'ns' is not empty`?

Reply:
I guess the deletion / check is recursive?

Reply:
> I wonder if we couldn't simplify the message to just `namespace 'ns' is not empty`?

wfm